Core/include/Acts/Seeding/SeedFinder.hpp

```diff
@@ -115,8 +113,8 @@ class SeedFinder {
   void getCompatibleDoublets(
```

Comment:
Even if this is private I think it wouldn't be a bad idea to had comments to define what are the function input, output and what it does.